tests/commands/test_commands.py

```diff
@@ -797,8 +797,9 @@ def test_download_data_timerange(mocker, markets):
     start_download_data(pargs)
     assert dl_mock.call_count == 1
     # 20days ago
-    days_ago = dt_floor_day(dt_now() - timedelta(days=20)).timestamp()
-    assert dl_mock.call_args_list[0][1]["timerange"].startts == days_ago
+    days_ago = datetime.now() - timedelta(days=20)
+    days_ago = dt_utc(days_ago.year, days_ago.month, days_ago.day)
+    assert dl_mock.call_args_list[0][1]["timerange"].startts == days_ago.timestamp()
```

Comment:
I'm not entirely convinced if this fixes it - or if this introduces a bug in different timezones instead (or covers up a bug in the code)

It's aligning the test to the code - but should the code really use `datetime.now()`? 
I'll leave this "hanging" for now until i've had some more time to think about it ... :laughing:

Reply:
Using datetime.now() is reasonable since we generally think in our local timezone, however, we use UTC finally ...

Reply:
works for me
i've enhanced the test-case a bit to ensure this is working in several different timezones (yes, it expands this test to actually 20 test cases - but i think they run quick enough for that) :laughing: